src/com/estafet/learning/US005/Task5.java

```diff
@@ -0,0 +1,142 @@
+                reader.next();
+            }
+            number = reader.nextInt();
+        } while (number <= 0);
+
+        int n1 = 0, n2 = 1, n3, i;
+        System.out.print(n1 + " " + n2);
+
+        for (i = 2; i < number; ++i) {
+            n3 = n1 + n2;
+            System.out.print(" " + n3);
+            n1 = n2;
+            n2 = n3;
+        }
+    }
+
+    public static void factorial() {
+        Scanner reader = new Scanner(System.in);
+        System.out.format("%nPlease enter a positive number to view Factorial of it: %n");
+
+        do {
+            while (!reader.hasNextInt()) {
+                System.out.format("That's not a number!%n");
+                System.out.format("Enter a positive number to view Factorial of it: %n");
+                reader.next();
+            }
+            number = reader.nextInt();
+        } while (number <= 0);
+
+        long i, fact = 1;
+        for (i = 1; i <= number; i++) {
+            fact = fact * i;
+        }
+        System.out.format("Factorial of " + number + " is: " + fact + "%n");
+    }
+
+    public static void factorialOfOddNumbers() {
+        Scanner reader = new Scanner(System.in);
+        System.out.format("Please enter a positive and odd number to view Factorial of it: %n");
+        number = reader.nextInt();
+
+        if (number > 0 && number % 2 != 0) {
+            long i, fact = 1;
+            for (i = 1; i <= number; i++) {
+                fact = fact * i;
+            }
+            System.out.format("Factorial of " + number + " is: " + fact + "%n");
+        } else {
+            System.out.format("Please enter a positive and odd number to view Factorial of it: %n");
+        }
+    }
+
+    public static void addTwoNumbers() {
+        Scanner reader = new Scanner(System.in);
+        do {
+            System.out.format("Please enter two numbers to calculate the sum of them: %n");
+            int numberOne = reader.nextInt();
+            int numberTwo = reader.nextInt();
+            int sum = numberOne + numberTwo;
+            System.out.format("The sum of these numbers " + numberOne + " and " + numberTwo + " is : " + sum + "%n");
+
+            System.out.format("Would you like to do this operation again? ");
+        }
+        while (!reader.next().equals("no"));
+    }
+
+    public static void numberOfDaysInMonth() {
+        Scanner input = new Scanner(System.in);
+        int number_Of_DaysInMonth = 0;
+
+        System.out.format("Input a month number between 1 and 12: %n");
+        int month = input.nextInt();
+        if (month < 1 || month > 12) {
+            System.out.format("This is not a valid month.%n");
+        }
+
+        System.out.format("Input a year: ");
+        int year = input.nextInt();
+        switch (month) {
+            case 1:
+            case 3:
+            case 5:
+            case 12:
+            case 10:
+            case 8:
+            case 7:
```

Comment:
don't really like the idea from (Oracle example) [https://docs.oracle.com/javase/tutorial/displayCode.html?code=https://docs.oracle.com/javase/tutorial/java/nutsandbolts/examples/SwitchDemo2.java]. Consider someone adds some assignment for a `case:` above one, that what you expected would assign a value, e.g. if I set `case 5: number_Of_DaysInMonth = 10;` what will be the value if `month = 3`

Reply:
changed logic